Framework/Algorithms/src/CompareWorkspaces.cpp

```diff
@@ -1109,10 +1109,10 @@ void CompareWorkspaces::doPeaksComparison(PeaksWorkspace_sptr tws1,
       std::string name = col->name();
       double s1 = 0.0;
       double s2 = 0.0;
-      if (name == "runnumber") {
+      if (name == "RunNumber") {
```

Comment:
Thanks for catching and fixing this error. Should we try and protect against further issues in the future by transforming `name` to lower case first and then comparing with all lowercase?